tests/testsuite.py

```diff
@@ -76,6 +76,14 @@ class ActiniaTestCase(unittest.TestCase):
     user = None
     auth_header = {}
     users_list = []
+    project_url_part = "projects"
```

Comment:
For the plugin, `project_url_part` were imported from actinia-core (if no circular reference).
Is this also possible here?